torcheeg/trainers/classifier.py

```diff
@@ -98,6 +115,14 @@ def init_metrics(self, metrics: List[str], num_classes: int) -> None:
         self.val_metrics = classification_metrics(metrics, num_classes)
         self.test_metrics = classification_metrics(metrics, num_classes)
 
+    def get_MissingParameter(self,dataloader):
```

Comment:
You can trust the parameters provided by the user, but never trust the data input by the user, including its shape. This can lead to nearly unreadable error messages and unexpected bugs.